### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by Identidy** [1kc/razer-macos#222]
my Razer blackwidow elite and my razer deathadder elite dont work, nothing on the menu thing pops up apart from, refresh device list spectrum all devices and cycle all devices. is there a fix?

**Comment by Filthic** [comment]
Same thing happens with Razer Basilisk v2

**Comment by Identidy** [comment]
> Same thing happens with Razer Basilisk v2

yeah, this keyboard is meant to work with it aswell, really annoying.

**Comment by wortkrieg** [comment]
Same here. Razer Basilisk v2. MacBook Pro 16" Intel.

**Comment by joeyiscool12** [comment]
Same, razer viper ultimate, even though its confirmed supported

**Comment by Bryghton42** [comment]
> my Razer blackwidow elite and my razer deathadder elite dont work, nothing on the menu thing pops up apart from, refresh device list spectrum all devices and cycle all devices. is there a fix?

You need to change the on board profile for it!

**Comment by dhobi** [comment]
Any news on this? Did the proposed fix from @Bryghton42 work for some of you?
